assets/sass/v2/_forms.scss

```diff
@@ -0,0 +1,6 @@
+.siw-main-view {
```

Comment:
- can you add same fix to webauthn.scss (wrap style inside `.siw-main-view`)
- the ideal fix would be in _all.scss
```
.siw-main-view {
  import ...
  import ..
}
```
and remove the nested style in each other file.

Reply:
I would check with Edbury on that and make a change. Just added a wrapper for webauthn for now.